testsuite/scripts/tutorials/test_02-charged_system-2.py

```diff
@@ -27,6 +28,22 @@
 class Tutorial(ut.TestCase):
     system = tutorial.system
 
+    def test_distribution(self):
+        """
+        checks if the particle distribution is within the box and
+        the mean of the distribution matches with a reference value
+        """
+        avg = 0
+        for i in range(1, 3):
+            pos = np.flatnonzero(tutorial.res[:, i] > 0)
+            self.assertGreater(tutorial.res[pos[0], 0], tutorial.wall_margin)
+            self.assertLess(tutorial.res[pos[-1], 0],
+                            tutorial.box_z - tutorial.wall_margin)
+
+            avg += np.mean(tutorial.res[pos, i])
+        avg *= 0.5
+        self.assertAlmostEqual(avg, 0.0148, delta=5e-4)
```

Comment:
It is the mean particle density per bin of the histogram which i originally planned to compare against each bin value, which is not possible without sufficient sampling. I just noticed that this mean value is constant anyways, which is only fluctuating because i neglected the 0s. I will remove this comparison since it does not give any insight into the system.